### ISSUE TITLE
Install issues on macOS Sonoma

### ISSUE THREAD

**Issue by cronus130** [oznu/ffmpeg-for-homebridge#64]
**Describe The Bug:**
Install ffmpeg via terminal for macOS Sonoma per documentation:

`sudo curl -Lf# https://github.com/homebridge/ffmpeg-for-homebridge/releases/latest/download/ffmpeg-darwin-$(uname -m).tar.gz | sudo tar xzfm - -C / --no-same-owner`

**To Reproduce:**
Check ffmpeg install status with terminal command `ffmpeg`

**Expected behavior:**
Return ffmpeg version info

**ffmpeg output:**

``` 
dyld[2552]: Library not loaded: /usr/local/opt/libxcb/lib/libxcb.1.dylib
  Referenced from: <*****> /usr/local/bin/ffmpeg
  Reason: tried: '/usr/local/opt/libxcb/lib/libxcb.1.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OS/usr/local/opt/libxcb/lib/libxcb.1.dylib' (no such file), '/usr/local/opt/libxcb/lib/libxcb.1.dylib' (no such file), '/usr/local/lib/libxcb.1.dylib' (no such file), '/usr/lib/libxcb.1.dylib' (no such file, not in dyld cache)
zsh: abort      ffmpeg
```

**Environment:**
Intel Mac Mini 2018

**Operating System**: 
macOS Sonoma 14.0

**Comment by akreis** [comment]
The Mac OS install command does create /usr/local/bin/ffmpeg in the PATH

However, it does not create
/usr/local/opt/libxcb/lib/libxcb.1.dylib
or
/usr/local/lib/libxcb.1.dylib
or
/usr/lib/libxcb.1.dylib

ffmpeg output:

dyld[9305]: Library not loaded: '/usr/local/opt/libxcb/lib/libxcb.1.dylib'
  Referenced from: '/usr/local/bin/ffmpeg'
  Reason: tried: '/usr/local/opt/libxcb/lib/libxcb.1.dylib' (no such file), '/usr/local/lib/libxcb.1.dylib' (no such file), '/usr/lib/libxcb.1.dylib' (no such file)
zsh: abort      ffmpeg

[10/2/2023, 8:47:10 PM] [homebridge-unifi-protect] Error running ffmpeg: Command failed: ffmpeg -hide_banner -codecs
dyld[6608]: Library not loaded: '/usr/local/opt/libxcb/lib/libxcb.1.dylib'
  Referenced from: '/usr/local/bin/ffmpeg'
  Reason: tried: '/usr/local/opt/libxcb/lib/libxcb.1.dylib' (no such file), '/usr/local/lib/libxcb.1.dylib' (no such file), '/usr/lib/libxcb.1.dylib' (no such file)

[10/2/2023, 8:47:10 PM] [homebridge-unifi-protect] Unable to probe the capabilities of your Homebridge host without access to 'ffmpeg'. Ensure that it is available in your path and correctly working.


Environment:
2013 Mac Pro Intel 12 core

Operating System:
macOS Monterey 12.7

**Comment by hjdhjd** [comment]
Thanks for the reports…I’ll investigate later in the week. Don’t need anything more on this one in the meantime.

**Comment by hjdhjd** [comment]
Addressed in v2.1.1
